DEVELOPING.md

```diff
@@ -102,24 +99,12 @@ For more information, please see https://flow.org/en/docs/react/components/#toc-
 Install the following plugins in your preferred editor
 
 - Editor Config (standardizes basic editor configuration)
-- ESLint (Javascript linting)
+- ESLint (JavaScript linting)
 - Stylelint (CSS linting)
-- Prettier (Javscript formatting)
+- Prettier (JavaScript formatting)
 - Sass (Stylesheets)
 - Babel (Transpiler)
 
-## Testing Embedded UI Elements With Your Data
```

Comment:
We decided that the /examples folder isn't going to be a priority. Rather than update the instructions with storybook (but have the examples still broken), we're just going to remove the docs here and clean up the /examples directory as a chore.